Assets/MixedRealityToolkit.Tests/PlayModeTests/DemoSceneTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
```

Comment:
Take a look at the other play mode tests - we end up building only in non-UWP because of weirdness with .NET. (So you'll need to add that #ifdef chunk here as well)